File: 6-Arrays/secondQuestion.js
function printElement(array, index) {
    if(index < array.length) {
        console.log(array[index]);
    } else {
        console.log('index out of bounds');
    }
}

const example = [222,333,444];
printElement(example, 0);
printElement(example, 3);

File: 06-Arrays/SecondQuestion.java
public class SecondQuestion {
    public static void main(String[] args) throws Exception {
        int[] example = {2,3,4,5};
        printElement(example, 5);
    }

    public static void printElement(int[] array, int index) {
        if(index < array.length) {
            System.out.println(array[index]);
        } else {
            System.out.println("Index out of bounds");
        }
    }
}
